homeassistant/components/knx/schema.py

```diff
@@ -34,6 +34,7 @@
     vol.All(vol.Coerce(int), vol.Range(min=1, max=65535)),
     msg="value does not match pattern for KNX group address '<main>/<middle>/<sub>', '<main>/<sub>' or '<free>' (eg.'1/2/3', '9/234', '123')",
 )
+ga_or_list_validator = vol.Any(ga_validator, [ga_validator])
```

Comment:
Any reason why this does not ensure a list? That way, the output from the config is always a list (and removes the need for the rest of the code to handle bot string and list cases.

```suggestion
ga_or_list_validator = vol.All(cv.ensure_list, [ga_validator])
```

Reply:
No specific reason. It is just passed to xknx - and this handles both for backwards compatibility. Could just as well pack it into a list and extract it in xknx again...
https://github.com/XKNX/xknx/blob/87d20e373696ba49c062a4297545cdc0f8f9726e/xknx/remote_value/remote_value.py#L36